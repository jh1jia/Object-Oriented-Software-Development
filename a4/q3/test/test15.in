sample.txt dropfirst 3 count l doublewords doublewords doublewords count 2 count 1 allcaps count 3
