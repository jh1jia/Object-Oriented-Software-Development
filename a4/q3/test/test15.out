1O
2O
1O
4O
5O
6O
7O
8O
9D
2D
2D
4D
5D
6D
7D
8D
